pom.xml

```diff
@@ -54,6 +56,12 @@
 			<artifactId>lombok</artifactId>
 			<optional>true</optional>
```

Comment:
If we are doing pom cleanup, we should define the lombok version and make it compile only, since the version heavily depends on the java version.

Reply:
lombok version is defined in parent pom and optional is a valid/better [alternative to provided](https://stackoverflow.com/questions/40393098/when-to-use-optionaltrue-optional-and-when-to-use-scopeprovided-scope) in this case.